tests/conftest.py

```diff
@@ -0,0 +1,197 @@
+import os
+import re
+import signal
+import socket
+import subprocess
+import sys
+
+import pytest
+
+# ==============================================================================
+
+WORKING_DIR = os.path.dirname(os.path.abspath(__file__)) + '/'
+
+REG_NBD_PATH = re.compile("^NBD `(/dev/nbd[0-9]+)` is now attached.$")
+
+HTTP_PORT = '8080'
+
+# ==============================================================================
+# Helpers.
+# ==============================================================================
+
+class TimeoutException(Exception):
+    pass
+
+def timeout_call(timeoutseconds, function, *arguments):
+    def handler(signum, frame):
+        raise TimeoutException()
+    signal.signal(signal.SIGALRM, handler)
+    signal.alarm(timeoutseconds)
+    try:
+        result = function(*arguments)
+        signal.alarm(0)
+        return result
+    except:
+        signal.alarm(0)
+        raise
```

Comment:
```suggestion
    try:
        return function(*arguments)
    finally:
        signal.alarm(0)
```

Reply:
I know you read the other PR. Isn'it? :)

Reply:
yup :p